configs/tsconfig-compass/tsconfig.common.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@mongodb-js/tsconfig-devtools/tsconfig.common.json",
   "compilerOptions": {
-    "removeComments": false
+    "removeComments": false,
+    "moduleResolution": "node16"
```

Comment:
Moved this from the compass-preferences-model config to the shared one, don't know if there's any harm in it but generally we should be able to support `import { ... } from 'package/export'` in all our tooling afaik